addons/sourcemod/scripting/nominations_extended.sp

```diff
@@ -134,6 +143,10 @@ public void OnLibraryAdded(const char[] szName)
 	{
 		g_bKzTimer = true;
 	}
+	if (StrEqual (szName, "gokz-core"))
+	{
+		g_bKzTimer = true;

```

Comment:
GOKZ should be on its own boolean - returning that KZTimer exists breaks GOKZ usability with other parts of this plugin.